src/artist-selection/artists-slider.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {ChangeEvent, useContext, useState} from "react";
+import {observer} from "mobx-react";
+import {LocaleContext} from "../spotifun";
+import "./artists-slider.css"
+
+export const ArtistsSlider: React.FunctionComponent<ArtistsSliderProps> = observer(({onSliderChange}) => {
+
+    const [recommendationLimit, setRecommendationLimit] = useState<number>(50);
+    // const strings = useContext(LocaleContext);
+
+    const onChange = (e: ChangeEvent<HTMLInputElement>) => {
+        setRecommendationLimit(e.target.valueAsNumber);
+        onSliderChange(e);
+    }
+
+    return (
+        <div className="slidecontainer">
```

Comment:
Are you sure this outer container is needed? The entire page should already be centered

Reply:
It's not centered without it